ethers-core/src/types/transaction/eip2930.rs

```diff
@@ -0,0 +1,24 @@
+use crate::types::{Address, H256};
+
+use rlp_derive::RlpEncodable;
+use serde::{Deserialize, Serialize};
+
+/// Access list
+#[derive(Debug, Default, Clone, PartialEq, Serialize, Deserialize, RlpEncodable)]
```

Comment:
I think I hit one bug here, `RlpEncodable` should be `RlpEncodableWrapper`.  unless you're using `.0` to do the rlp encoding, otherwise you got an extra "[]".